app/src/main/java/com/greenfox/tribesoflagopusandroid/LoginActivity.java

```diff
@@ -41,7 +44,7 @@ public void login(View view) {
         Intent intent = new Intent(this, MainActivity.class);
         startActivity(intent);
 
-        LoginService service = ServiceFactory.createMockService(LoginService.class, "https://tribes-of-lagopus.herokuapp.com/");
+        LoginService service = ServiceFactory.createMockService();
```

Comment:
@georgezsiga this is why we inject, so we dont need to create this here, at this point there should be an injected `LoginService`